### PR TITLE
removed unnecessary interface declarations from gameplay

### PR DESCRIPTION
**Pull request by hdemusg**
## Description
Removing unused interfaces from gameplay Python files. 

## Associated Issue
'remove "custom" interfaces' on Clickup

## Steps to test
Run soccer to make sure it works

Expected result: It should run

### CODE REVIEW

File: rj_gameplay/rj_gameplay/skill/move.py [hunk 1]
```diff
@@ -16,14 +16,15 @@
...
 from rj_msgs import msg
 
-class IMove(skill.ISkill, ABC):
-    ...
-
-
 """
 A skill version of move so that actions don't have to be called in tactics
 """
+<<<<<<< HEAD
```
Comment (kfu02) [inline rj_gameplay/rj_gameplay/skill/move.py]: fix this merge conflict

File: rj_gameplay/rj_gameplay/skill/pivot_kick.py [hunk 2]
```diff
@@ -68,8 +63,12 @@ def tick(self, robot: rc.Robot, world_state: rc.WorldState) -> RobotActions:
         return actions
 
     def is_done(self, world_state: rc.WorldState) -> bool:
+<<<<<<< HEAD
```
Comment (kfu02) [inline rj_gameplay/rj_gameplay/skill/pivot_kick.py]: fix this merge conflict

File: rj_gameplay/rj_gameplay/skill/stub.py [hunk 3]
```diff
@@ -2,7 +2,6 @@
 
 import stp.skill as skill
 
-
 class Stub(skill.ISkill):
```
Comment (kfu02) [inline rj_gameplay/rj_gameplay/skill/stub.py]: remove this file please

**Review by kfu02 — CHANGES_REQUESTED** [review]
please make sure this builds

**Review by RyanBard7 — APPROVED** [review]

**Review by Yudai-8-om — APPROVED** [review]
LGTM!

**Review by anshulm07 — APPROVED** [review]
Looks good.

**Pull request merged**